src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days > 6) {
+    return days * 40 - 50;
```

Comment:
checklist 1st https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md?rgh-link-date=2022-06-06T07%3A07%3A40Z
You need to create some variables for the main values.